simpleprophet/simpleprophet/pipeline.py

```diff
@@ -11,26 +11,39 @@
 from google.cloud import bigquery
 import pandas as pd
 
-from simpleprophet.output import reset_output_table, write_forecasts, prepare_records, write_records
+from simpleprophet.output import reset_output_table, write_forecasts
+from simpleprophet.output import prepare_records, write_records
```

Comment:
Style note: I would expect to see a long import like this styled with parentheses:

```
from simpleprophet.output import (reset_output_table, write_forecasts, 
                                  prepare_records, write_records)
```